src/main/webapp/app/entities/skill/skill-update.component.html

```diff
@@ -64,7 +64,7 @@ <h2 id="jhi-skill-heading" jhiTranslate="teamdojoApp.skill.home.createOrEditLabe
                 <div class="form-group">
                     <label class="form-control-label" jhiTranslate="teamdojoApp.skill.expiryPeriod" for="field_expiryPeriod">Expiry Period</label>
                     <input type="text" class="form-control" name="expiryPeriod" id="field_expiryPeriod"
-                        [(ngModel)]="skill.expiryPeriod" pattern="^P(?:([-+]?[0-9]+)Y)?(?:([-+]?[0-9]+)M)?(?:([-+]?[0-9]+)W)?(?:([-+]?[0-9]+)D)?$"/>
+                        [(ngModel)]="skill.expiryPeriod" pattern="^[0-9]*"/>
```

Comment:
Isn't it better to change the expiryPeriod in then ```skill.model.ts``` to a number? That way we could use a input with ```type="number"``` and could leave the number validation to the browser (and let him handle edge cases like numbers in special languages).

Reply:
@jensvoss0253 already tried to use number as the fieldtype, without success.

Reply:
Somehow validating the input as a number field did not work - the (somewhat expanded) regex does the job equally well, though.